app/src/main/java/at/aau/serg/models/LobbyPlayer.kt

```diff
@@ -0,0 +1,5 @@
+package at.aau.serg.models
+
+
+data class LobbyPlayer(var uuid: String, var name: String, var isVisible: Int) {
```

Comment:
Use an enum for `isVisible` since it is confusing if you don't know what visibility which number corresponds to.